snapcraft/internal/deprecations.py

```diff
@@ -31,6 +31,8 @@
     'dn4': "The 'history' command has been replaced by 'list-revisions'.",
     'dn5': "Aliases are now handled by the store, and shouldn't be declared "
            "in the snap.",
+    'dn6': "Use of the 'snap' command with a directory has been deprecated "
+           "in favour of the pack command.",
```

Comment:
All of the commands in this list are quoted except for `pack`.

Reply:
quoting is indeed something good to have